astropy/table/index.py

```diff
@@ -765,35 +765,26 @@ def __getitem__(self, item):
 
 
 class TableLoc:
-    '''
+    """
     A pseudo-list of Table rows allowing for retrieval
     of rows by indexed column values.
 
     Parameters
     ----------
     table : Table
         Indexed table to use
-    '''
+    """
 
     def __init__(self, table):
         self.table = table
         self.indices = table.indices
         if len(self.indices) == 0:
             raise ValueError("Cannot create TableLoc object with no indices")
 
-    def __getitem__(self, item):
-        '''
-        Retrieve Table rows by value slice.
-
-        Parameters
-        ----------
-        item : column element, list, ndarray, slice or tuple
-            Can be a value of the table primary index, a list/ndarray
-            of such values, or a value slice (both endpoints are included).
-            If a tuple is provided, the first element must be
-            an index to use instead of the primary key, and the
-            second element must be as above.
-        '''
+    def _get_rows(self, item):
+        """
+                Retrieve Table rows indexes by value slice.
```

Comment:
Align with quotes.

Reply:
actually @taldcroft said to double quotes the doc strings according  to pep8.
should i change it?

Reply:
Double quotes is fine, but the text should be aligned.